src/lib/controllers/configuration-manager/generators/generate-ca-cert.ts

```diff
@@ -0,0 +1,62 @@
+import * as forge from 'node-forge';
+import { convertCertAttrs } from '.';
```

Comment:
Bit confused about this. Are we defining definition files internally? If so, why?

Reply:
A certMap is far more dense/readable and repetitive than:
```
{
shortName: key,
value: Value,
}
```
when writing configuration/config-manifests. You may also check the code below.